cleaner_test.go

```diff
@@ -1,20 +1,36 @@
 package rmq
 
 import (
+	"os"
 	"testing"
 	"time"
 
+	"github.com/alicebob/miniredis/v2"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
+func testRedis(t testing.TB) (addr string, closer func()) {
```

Comment:
What do you think if we put that func at the bottom of the test file?

Reply:
What's the rationale for such layout?

Reply:
> What's the rationale for such layout?

Unexported helpers functions are usually put at the end of file, since they are just helpers and not the main purpose of the file.

There is a layout approach I really like, I think I will share with the team later in Q1 to discuss.